### PR TITLE
Added missing 'protos' directory to package.json.

### LINKED ISSUE #11: Error: google/monitoring/v3/metric_service.proto could not be found
I get the following error when attempting to run the samples:

```
C:\git\nodejs-monitoring\samples\node_modules\yargs\yargs.js:1057
      else throw err
           ^

Error: google/monitoring/v3/metric_service.proto could not be found in C:\git\nodejs-monitoring\samples\node_modules\@google-cloud\monitoring\protos
    at Function.GrpcClient._resolveFile (C:\git\nodejs-monitoring\samples\node_modules\google-gax\lib\grpc.js:172:9)
    at GrpcClient.loadProto (C:\git\nodejs-monitoring\samples\node_modules\google-gax\lib\grpc.js:161:33)
    at new MetricServiceClient (C:\git\nodejs-monitoring\samples\node_modules\@google-cloud\monitoring\src\v3\metric_service_client.js:94:15)
    at listMetricDescriptors (C:\git\nodejs-monitoring\samples\metrics.js:90:18)
    at Object.require.demand.command.command.opts [as handler] (C:\git\nodejs-monitoring\samples\metrics.js:541:5)
    at Object.runCommand (C:\git\nodejs-monitoring\samples\node_modules\yargs\lib\command.js:226:22)
    at Object.parseArgs [as _parseArgs] (C:\git\nodejs-monitoring\samples\node_modules\yargs\yargs.js:970:30)
    at Object.get [as argv] (C:\git\nodejs-monitoring\samples\node_modules\yargs\yargs.js:912:21)
    at Object.<anonymous> (C:\git\nodejs-monitoring\samples\metrics.js:623:22)
    at Module._compile (module.js:624:30)

```

### PR DESCRIPTION
**Pull request by djbelcher**
Fixes #11

- [X] Tests and linter pass
- [X] Code coverage does not decrease (if any source code was changed)
- [X] Appropriate docs were updated (if necessary)

### CODE REVIEW

**Comment by djbelcher** [comment]
I signed it!

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@google-cloud/monitoring",
   "description": "Stackdriver Monitoring API client for Node.js",
-  "version": "0.4.0",
+  "version": "0.4.1",
```
Comment (stephenplusplus) [inline package.json]: Please remove this change. We handle the releases in separate PRs.

**Comment by stephenplusplus** [comment]
Thank you very much for this!

**Comment by stephenplusplus** [comment]
Thanks! Release coming.

**Pull request merged**